fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -176,6 +191,14 @@ public RecordBuiltRangeException(@Nullable Tuple start, @Nullable Tuple end) {
         }
     }
 
+    /**
+     * Get the current number of records to process in one transaction.
+     * @return the current number of records to process in one transaction
```

Comment:
Might be worth mentioning in this Javadoc description that this can go up or down during program execution.